two_factor/models.py

```diff
@@ -87,11 +87,50 @@ def verify_token(self, token):
         except ValueError:
             return False
 
-        for drift in range(-5, 1):
+        for drift in self.drift_range:
             if totp(self.bin_key, drift=drift, digits=totp_digits()) == token:
                 return True
         return False
 
+    def get_throttle_factor(self):
+        return getattr(settings, 'TWO_FACTOR_PHONE_THROTTLE_FACTOR', 1)
```

Comment:
This should not be in the mixin and there should be a `TWO_FACTOR_EMAIL_THROTTLE_FACTOR` setting for `EmailDevice`